oaknational/oak-ai-lesson-assistant
# build: release candidate

## Description

- List of changes

## Issue(s)

Fixes #

## How to test

1. Go to https://oak-ai-lesson-assistant-lqv47un94.vercel-preview.thenational.academy
2. Click on \_\_\_\_\_\_
3. You should see \_\_\_\_\_\_

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
